runtime/onert/backend/train/ops/PoolLayer.cc

```diff
@@ -58,8 +58,10 @@ class MaxPool2D final : public TrainingKernelRegistry
       _op_params.stride_width = strideWidth;
       _op_params.filter_height = kernelHeight;
       _op_params.filter_width = kernelWidth;
-      _op_params.padding_values.height = (int8_t)paddingTop;
-      _op_params.padding_values.width = (int8_t)paddingLeft;
+      assert(paddingTop < (1 << 8));
+      assert(paddingLeft < (1 << 8));
+      _op_params.padding_values.height = static_cast<uint8_t>(paddingTop);
+      _op_params.padding_values.width = static_cast<uint8_t>(paddingLeft);
```

Comment:
Ah? It looks like typecasting to `int16_t` is necessary, not `uint8_t`.  

https://github.com/Samsung/ONE/blob/c82f8cbc5ef0faa70d5b0ebf9dfd53020a1fded9/compute/cker/include/cker/Types.h#L81-L83

https://github.com/Samsung/ONE/blob/c82f8cbc5ef0faa70d5b0ebf9dfd53020a1fded9/compute/cker/include/cker/Types.h#L66-L70

Reply:
Tthere is no problem since `_op_param.padding_values` are filled with zero-extended value from positive numbers of equal range.
- These padding parameters are always positive number.
- The positive number range of `int16_t` is equal to the range of `uint8_t`.

Reply:
Aha I got it! Thank you for explanation :+1:

Reply:
> The positive number range of int16_t is equal to the range of uint8_t. 

@ragmani Sorry, But I have another question. 
Does it mean that `int16_t` and `uint8_t` have a same max value?

Reply:
Sorry, it was not wrong information. They are not the same max value.